internal/pkg/cli/dialog/create_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
-	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/naming"
```

Comment:
2 a dalsie commits su nasledne zmeny v kode, z nazvu commitu by to malo byt jasne.